src/clock/Clock.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+let interval;
+
+export class Clock extends React.Component {
+  state = {
+    time: new Date().toLocaleTimeString(),
+  }
+
+  componentDidMount() {
+    interval = setInterval(() => {
+      const date = new Date().toLocaleTimeString();
+
+      // eslint-disable-next-line
+      console.log(date);
+      this.setState({ time: date });
+    }, 1000);
+  }
+
+  componentDidUpdate(prevProps) {
+    const { name } = this.props;
+
+    if (name === prevProps.name) {
```

Comment:
```suggestion
    if (name !== prevProps.name) {
```

https://user-images.githubusercontent.com/38065883/120624232-279ad200-c469-11eb-84d2-c3f5aff3e305.mov

Staren'kyi, clock wasn't renamed

Reply:
f*ck, dummy mistake, fixed